dom.go

```diff
@@ -200,8 +200,13 @@ func (h *HTML) reconcile(prev *HTML) []Mounter {
 		}
 	}
 
+	new := h.node != prev.node
 	// Properties
 	for name, value := range h.properties {
+		if new {
+			h.node.Set(name, value)
+			continue
+		}
```

Comment:
Why do this here instead of below (line 214) i.e. checking `if value != oldValue` to become `if new || value != oldValue` ?

Also, doesn't this mean the Attributes, Dataset, and Styles sections below are also all invalid since they are effectively the same logic?

Reply:
> Why do this here instead of below

Just slightly cheaper

> Also, doesn't this mean...

Seems likely, I'll take another pass at this tonight.

Reply:
> Just slightly cheaper

Got it; I would suggest maybe using the `if` form I suggested above purely for readability. But I am OK either way / do not feel too strongly about this.